ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -262,17 +241,16 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		// the RayJob is submitted against the RayCluster created by THIS job, then
 		// try to gracefully stop the Ray job and delete (suspend) the cluster
 		if rayJobInstance.Spec.Suspend && len(rayJobInstance.Spec.ClusterSelector) == 0 {
-			info, err := rayDashboardClient.GetJobInfo(ctx, rayJobInstance.Status.JobId)
```

Comment:
We don't need to have an additional GetJobInfo.

Reply:
Good catch!